BuberBreakfast/Data/DataContext.cs

```diff
@@ -0,0 +1,15 @@
+using BuberBreakfast.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace BuberBreakfast.Data
+{
+    public class DataContext : DbContext
+    {
+        public DataContext(DbContextOptions<DataContext> options) : base(options)
+        {
+
+        }
+
+        public DbSet<Shop> Shops { get; set; }
```

Comment:
# Remark
syntax is  
public DbSet<`model-name`> `can-be-anything-tho-recommended-to-be-same-as-model` { get; set; }